source/includes/_users.md

```diff
@@ -300,3 +300,153 @@ Delete a user.
 
 ### Response
 A `User` resource.
+
+
+
+
+## Set user password
+
+> Request Example
+
+```shell
+$ curl -X POST -H "Content-Type: application/json" -H "tenant: <tenant-name>" -u user:password -d '{"password": <new-password>}' "http://<manager-ip>/api/v3/users/<user-name>"
+```
+
+```python
+# Python Client-
+client.users.set_password(<user-name>, <new-password>)
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64'),
+   'tenant': <tenant-name>
+}
+
+vat data = {
+    "password": <new-password>,
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/users/<user-name>",
+  "method": "POST",
+  "headers": headers,
+  "data": data
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+    "username": "user",
+    "last_login_at": "2017-01-22T15:09:33.799Z",
+    "role": "user",
+    "groups": [],
+    "active": true,
+    "tenants": ["default_tenant"]
+}
+```
+
+`POST -d '{"password": <new-password>}' '"{manager-ip}/api/v3/users/{user-name}"`
+
+Delete a user.
+
+### URI Parameters
+* `user-name`: The name of the user to change his password.
```

Comment:
The name of the user **whose password is to be changed.** ~~to change his password.~~